x-pack/solutions/observability/plugins/observability/server/lib/rules/custom_threshold/custom_threshold_executor.ts

```diff
@@ -281,7 +281,10 @@ export const createCustomThresholdExecutor = ({
 
         const indexedStartedAt = start ?? startedAt.toISOString();
         scheduledActionsCount++;
-
+        const dataViewIdTitle =
+          typeof params.searchConfiguration?.index === 'string'
+            ? params.searchConfiguration?.index
+            : params.searchConfiguration?.index?.title;
```

Comment:
in this case, we will pass dataViewId for both ad-hoc and saved data view, right? Can we add a check here that will make sure we only pass dataViewId if it's a saved data view?